### ISSUE TITLE
renderGrViz unused argument (env = env)

### ISSUE THREAD

**Issue by rvernica** [rich-iannone/DiagrammeR#373]
I'm using DiagrammeR in a Shiny app. The code stopped working recently I suspect is due to a library upgrade. It fails with this error:
```R
Warning: Error in htmlwidgets::shinyRenderWidget: unused argument (evn = env)
  50: renderGrViz
  49: server [...server.R#1050]
Error in htmlwidgets::shinyRenderWidget(expr = expr, outputFunction = grVizOutput,  : 
  unused argument (evn = env)
```
Does this ring a bell?

Session info:
<details>

   ```R
> sessionInfo()
R version 3.6.2 (2019-12-12)
Platform: x86_64-redhat-linux-gnu (64-bit)
Running under: Fedora 30 (Workstation Edition)

Matrix products: default
BLAS/LAPACK: /usr/lib64/R/lib/libRblas.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C               LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8     LC_MONETARY=en_US.UTF-8   
 [6] LC_MESSAGES=en_US.UTF-8    LC_PAPER=en_US.UTF-8       LC_NAME=C                  LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] stringr_1.4.0      magick_2.3         RColorBrewer_1.1-2 plyr_1.8.5         jsonlite_1.6       jose_1.0           openssl_1.4.1     
 [8] ggplot2_3.2.1      curl_4.3           DiagrammeR_1.0.5   scidb_2.0.0        bit64_0.9-7        bit_1.1-15.1       shiny_1.4.0       

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.3        compiler_3.6.2    pillar_1.4.3      later_1.0.0       tools_3.6.2       digest_0.6.23     lifecycle_0.1.0   tibble_2.1.3     
 [9] gtable_0.3.0      pkgconfig_2.0.3   rlang_0.4.3       rstudioapi_0.10   yaml_2.2.0        fastmap_1.0.1     withr_2.1.2       dplyr_0.8.3      
[17] htmlwidgets_1.5.1 askpass_1.1       tidyselect_1.0.0  grid_3.6.2        glue_1.3.1        data.table_1.12.8 R6_2.4.1          purrr_0.3.3      
[25] magrittr_1.5      scales_1.1.0      promises_1.1.0    htmltools_0.4.0   assertthat_0.2.1  mime_0.8          xtable_1.8-4      colorspace_1.4-1 
[33] httpuv_1.5.2      stringi_1.4.5     visNetwork_2.0.9  lazyeval_0.2.2    munsell_0.5.0     crayon_1.3.4  
   ```
</details>

**Comment by trafficonese** [comment]
Can you post a small example where that error occurs?

This minimal example works for me:
```
library(shiny)
library(DiagrammeR)

ui = fluidPage(grVizOutput('diagram' ))

server = function(input, output) {
  output$diagram <- renderGrViz({
    grViz("digraph {A;}")
  })
}

shinyApp(ui = ui, server = server)
```

**Comment by rvernica** [comment]
This seems related https://stackoverflow.com/questions/59853592/unable-to-run-a-shiny-app-using-grvizoutput-and-rendergrviz

**Comment by trafficonese** [comment]
Indeed, the current CRAN version apparently has this bug. The dev version works fine again and I couldnt find a difference in the `renderGrViz `or `grVizOutput` code.

**Comment by YsoSirius** [comment]
Yes it was commit f9e4284. The arg name was `evn` which is wrong.

**Comment by rvernica** [comment]
Good find! That makes sense. I think this issue is not resolved. I'm looking forward for the next CRAN release.

**Comment by larspijnappel** [comment]
As I use this package quite extensively, I kept an even closer eye on a new release of DiagrammeR. However, it appears that this fix has not been released yet on CRAN (latest version 1.0.5 still has this issue). 
As a workaround, I downgraded to version 1.0.1 (which is working fine again) but I'd rather prefer to have all my packages to the latest version.  Are there any plans to release this fix any time soon?

**Comment by kmezhoud** [comment]
Same issue with 1.05. 
With Kubuntu return me the same error message:
`Warning: Error in htmlwidgets::shinyRenderWidget: argument inutilisé (evn = env)   56: DiagrammeR::renderGrViz   50: server [/home/kirus/R/x86_64-pc-linux-gnu-library/3.6/bioCancer/app/server.R#67] Error in htmlwidgets::shinyRenderWidget(expr = expr, outpu`

With OSX, it remains without error but frozen
Actually I am also working with Diagrammer 1.01.
Thanks

**Comment by jekriske-lilly** [comment]
Any update on this? New CRAN version? @rich-iannone 

Thanks

**Comment by rich-iannone** [comment]
@jekriske-lilly @larspijnappel @kmezhoud @rvernica Really sorry for the long wait on this. There were a number of additional fixes required at the same time (for tibble 3.0.0, dplyr 1.0.0) on top of the massive blunder I made earlier. But, just got word from CRAN that DiagrammeR 1.0.6 has been accepted.
